src/kibana/components/vislib/visualizations/tile_map.js

```diff
@@ -527,19 +531,18 @@ define(function (require) {
      * @return {String} hex color
      */
     TileMap.prototype.quantizeColorScale = function (count, min, max) {
-      var self = this;
       var reds5 = ['#fed976', '#feb24c', '#fd8d3c', '#f03b20', '#bd0026'];
       var reds3 = ['#fecc5c', '#fd8d3c', '#e31a1c'];
       var reds1 = ['#ff6128'];
-      var colors = self._attr.colors = reds5;
+      var colors = this._attr.colors = reds5;
 
       if (max - min < 3) {
-        colors = self._attr.colors = reds1;
+        colors = this._attr.colors = reds1;
       } else if (max - min < 25) {
-        colors = self._attr.colors = reds3;
+        colors = this._attr.colors = reds3;
       }
 
-      var cScale = self._attr.cScale = d3.scale.quantize()
+      var cScale = this._attr.cScale = d3.scale.quantize()
         .domain([min, max])
         .range(colors);
       if (max === min) {
```

Comment:
Can you fix the indenting above here - the lines should be flush with the `var cScale` line. Also, please add a blank line before the `if`